packages/jest-runner/src/runTest.ts

```diff
@@ -142,6 +142,13 @@ async function runTestInternal(
     docblockPragmas,
     testPath: path,
   });
+
+  if (typeof environment.getVmContext !== 'function') {
```

Comment:
should we warn if `runScript` is present? Might make it harder for a single test env to support multiple versions of Jest, bt also less confusing than "why isn't my code running"